src/transformState.js

```diff
@@ -61,8 +61,33 @@
  * @param {Object} state
  * @param {Object[]} transforms
  */
+
 function transformState(state, transforms) {
-  // write code here
+  const linkToStateObj = state;
+
+  transforms.forEach((operationObj) => {
+    if (operationObj.operation === 'addProperties') {
```

Comment:
Можно переписать через switch.

Suggested change 
```suggestion
   switch (operation) {
    case 'addProperties':
        break;
      default:
```